src/webrtc/call.ts

```diff
@@ -696,13 +696,34 @@ export class MatrixCall extends EventEmitter {
         this.setState(CallState.Ended);
     }
 
+    private shouldAnswerWithMediaType(
+        wantedValue: boolean, valueOfTheOtherSide: boolean, type: "audio" | "video",
```

Comment:
Because these are allowed to be undefined, but we can't use optionals:
```suggestion
        wantedValue: boolean | undefined, valueOfTheOtherSide: boolean | undefined, type: "audio" | "video",
```